orga/app/templates/components/routes/authenticated/campaigns/details/participants-tab.hbs

```diff
@@ -0,0 +1,46 @@
+<div class="participant-list__export">
+  <a class="button button--link" href="{{campaign.urlToResult}}" target="_blank" download>
+    Export résultats (.csv)
+  </a>
+</div>
+
+<div class="panel">
+  <div class="participant-list__header">
+    Liste des participants ({{participants.meta.rowCount}})
+  </div>
+  <div class="table">
+    <table>
+      <thead>
+        <tr>
+          <th>Nom</th>
+          <th>Prénom</th>
+          {{#if campaign.idPixLabel}}
+            <th>{{campaign.idPixLabel}}</th>
+          {{/if}}
+        </tr>
+      </thead>
+
+      {{#if participants}}
+        <tbody>
+        {{#each participants as |participant|}}
+          <tr>
+            <td>{{participant.user.lastName}}</td>
+            <td>{{participant.user.firstName}}</td>
+            {{#if campaign.idPixLabel}}
+              <td>{{participant.participantExternalId}}</td>
+            {{/if}}
+          </tr>
+        {{/each}}
+        </tbody>
+      {{/if}}
+    </table>
+
+    {{#unless participants}}
+      <div class="participant-list__no-participants">En attente de participants</div>
+    {{/unless}}
+  </div>
+</div>
+
+{{#if participants}}
+  {{pagination-control pagination=participants.meta paginationLink="authenticated.campaigns.details.participants"}}
+{{/if}}
```

Comment:
Par rapport à mon commentaire sur l'homogénéité des styles entre paramètres et participants, on pourrait par exemple déplacer le bouton d'export à l'intérieur de la première ligne du tableau.

+ No newline

Reply:
À voir avec Manu ? @ebarbier

Reply:
Sur la maquette le bouton est dans la navbar, tout à droite de Paramètres et Participants

Reply:
✅